controllers/articles.controller.js

```diff
@@ -16,3 +21,16 @@ exports.getArticleById = (request, response, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.getComments = (request, response, next) => {
+  // console.log(request.params);
+  const { article_id } = request.params;
+  Promise.all([fetchComments(article_id), fetchArticleById(article_id)])
+    .then(([comments, article]) => {
+      if (article) {
+        return response.status(200).send({ comments });
+      }
+      Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
Ideally need to be doing your rejecting in your models -but if it works and passes the missing test, then leave it